### DISCUSSION TITLE
【功能建议】希望加入分页功能

### DISCUSSION THREAD

**Discussion by mcnk328** [AlistGo/alist#257] [Ideas]
### Alist Version / Alist 版本

v2.0.3-0-gd6d2f52-dirty

### Describe the bug / 问题描述

在列出网盘文件时，如果文件过多则会集中显示在一个页面上，电脑访问或许还好，但是手机访问时会有点卡顿

**Comment by xhofe** [comment]
计划下下个版本加入

**Comment by xhofe** [comment]
Available in v2.0.5.
